test/libsolidity/smtCheckerTests/imports/duplicated_errors_1.sol

```diff
@@ -0,0 +1,19 @@
+==== Source: a.sol ====
+contract A {
+	uint x;
+}
+==== Source: b.sol ====
+import "a.sol";
+contract B is A {
+	function g() public view { assert(x > x); }
+}
+==== Source: c.sol ====
+import "b.sol";
+contract C is B {
+	function h(uint x) public pure { assert(x < x); }
+}
+// ====
+// SMTEngine: all
+// ----
+// Warning 6328: (b.sol:62-75): CHC: Assertion violation happens here.\nCounterexample:\nx = 0\n\nTransaction trace:\nB.constructor()\nState: x = 0\nB.g()
+// Warning 6328: (c.sol:68-81): CHC: Assertion violation happens here.\nCounterexample:\nx = 0\nx = 0\n\nTransaction trace:\nC.constructor()\nState: x = 0\nC.h(0)
```

Comment:
Actually, why do these have locations? Wasn't the error triggered by errors with different descriptions and no location?

Reply:
These are triggered by the assertions themselves, that's why they have locations.

Reply:
So there should be more errors here? Perhaps they did not get printed due to the bug in `warning()`?

Reply:
Why should there be more errors? I think this is correct.

Reply:
In that case I don't understand why the ICE got triggered in the first place.

Reply:
Good q, let me think

Reply:
Weird, I'm re-running the issue on develop and it doesn't happen anymore.

Reply:
Ahh ok it happens only when there are no solvers available